lib/src/components/NxTag/types.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import {ReactNode, ValidationMap, HTMLAttributes} from 'react';
+import * as PropTypes from 'prop-types';
+
+export const NX_TAG_COLORS =
+    ['light-blue', 'purple', 'pink', 'blue', 'red', 'green', 'orange', 'yellow', 'lime', 'indigo'] as const;
```

Comment:
Indigo is defunct right?

Reply:
> Indigo is defunct right?

Hmm I'm noticing it's still in the mockup.  Seems weird though since it's basically the same as the default.

Reply:
> > Indigo is defunct right?
> 
> Hmm I'm noticing it's still in the mockup. Seems weird though since it's basically the same as the default.

I think the mockups are now a little out of date. @dturner32

Reply:
the mockups in zeplin are up to date as of 10 min ago

Reply:
> I think the mockups are now a little out of date. @dturner32

Per a conversation in slack this morning it is being removed from the mockups